docs/layers/column-layer.md

```diff
@@ -180,10 +180,10 @@ The maximum outline width in pixels.
 
 ##### `material` (Object, optional)
 
-* Default: `new PhongMaterial()`
+* Default: `true`
+
+This is an object that contains material props for [lighting effect](/docs/effects/lighting-effect.md) applied on extruded polygons. Check [the lighting guide](/docs/developer-guide/using-lighting.md) for more details.
```

Comment:
- link directly to `using-lighting.md#constructing-a-material-instance`
- "for more details" -> "for configurable settings"